nucliadb_models/src/nucliadb_models/search.py

```diff
@@ -1140,6 +1144,31 @@ class MetadataExtensionStrategy(RagStrategy):
     )
 
 
+class ConversationalStrategy(RagStrategy):
```

Comment:
Can we put somewhere in the docs that this rag strategy is in beta?
Maybe in the description of the `rag_strategies` field